### ISSUE TITLE
Training collapse at about 188 epochs.

### ISSUE THREAD

**Issue by GeoffreyChen777** [MohsenZand/ObjectBox#12]
Hi, thanks for your great work!

I'm using your code on a custom dataset. I transformed my dataset into your required format and then train the model on it.

At the beginning of the training, everything was fine. But at about 188 epochs, I got this:

<img width="1188" alt="image" src="https://user-images.githubusercontent.com/14183213/193810385-258f4366-8512-495e-aadb-9ae5c01f2877.png">

Do you have any insights on this such as any parameters changes at this step?

Best wishes.

**Comment by MohsenZand** [comment]
Thank you!
You can start with a smaller initial learning rate (it's in the 'hyp.yaml'). Also, changing the learning rate at epoch ~ 150 would help. Some data augmentations are also possible. Despite being available in the code, they are not utilized in the reported results. So, check out the 'hyp.yaml' and 'flag_sets.py'.

**Comment by GeoffreyChen777** [comment]
Thanks!
